nucypher/cli/characters/ursula.py

```diff
@@ -159,12 +133,9 @@ def ursula(click_config,
             raise click.BadArgumentUsage('--network is required to initialize a new configuration.')
 
         if dev:
-            actions.handle_control_output(message="WARNING: Using temporary storage area",
-                                          color='yellow',
-                                          quiet=quiet,
-                                          json=click_config.json)
+            click_config.emitter(message="WARNING: Using temporary storage area", color='yellow')
 
-        if not config_root:  # Flag
+        if not config_root:                         # Flag
```

Comment:
Bunch of whitespace here, is this PEP8 compliant?

Reply:
Probably aligning with another comment column

Reply:
Yes, it is. But I wasn't aware if that's PEP8 compliant or not. IIRC, it kinda doesn't suggest this? Am I wrong?

Reply:
No you are not wrong - just a preference.  You dont like it? :-)